src/dotnet-roslyn-tools/Commands/PRTaggerCommand.cs

```diff
@@ -13,8 +13,7 @@ internal static class PRTaggerCommand
     private static readonly PRTaggerCommandDefaultHandler s_prTaggerCommandHandler = new();
 
     private static readonly Option<string> ProductName = new Option<string>(new[] { "--product-name", "-n" }, "Name of product (e.g. 'roslyn' or 'razor')") { IsRequired = true }
-        .FromAmong("roslyn", "razor");
-    private static readonly Option<string> ProductRepoPath = new(new[] { "--repo-path", "-p" }, "Path to product repo") { IsRequired = true };
+        .FromAmong("roslyn", "razor", "fsharp");
```

Comment:
I think I have some code for `AllProducts` or something in the vsbranchinfo command.. might be something to generalize? eg, pretty sure this can work for typescript too, with no changes

Reply:
I think I would like to see this become non-required. Allow it to be passed to limit for testing or other purposes, but the default behavior would be to spin through the products and look for changes.

Reply:
Yep, just changed it to be non-required 😄